src/app.js

```diff
@@ -342,12 +349,228 @@ function buildImageImporter() {
 
   $importButton.click(function(){
     activateToolItem($importButton);
-    paper.initImageImport();
+    paper.initImageImport({mode:'trace'});
```

Comment:
Exactly what's going on between the manual and automatic trace that pertains to the bugs I've noted in the main PR comments are likely related to how this has been refactored.